scripts/migration/lib.sh

```diff
@@ -218,14 +225,16 @@ check_config() {
     exit 1
   fi
 
-  for prefix in sc wc; do
-    if check_sops "${CK8S_CONFIG_PATH}/.state/kube_config_${prefix}.yaml"; then
-      CONFIG["${prefix}-kubeconfig"]="encrypted"
-    else
-      CONFIG["${prefix}-kubeconfig"]="unencrypted"
-    fi
+  for prefix in wc sc; do
+    if [[ "${cluster:-}" =~ ^($prefix|both)$ ]]; then
+      if check_sops "${CK8S_CONFIG_PATH}/.state/kube_config_$prefix.yaml"; then
+        CONFIG["$prefix-kubeconfig"]="encrypted"
+      else
+        CONFIG["$prefix-kubeconfig"]="unencrypted"
+      fi
 
-    log_info "using ${prefix} kubeconfig ${CONFIG["${prefix}-kubeconfig"]}"
+      log_info "using $prefix kubeconfig ${CONFIG["$prefix-kubeconfig"]}"
+    fi
```

Comment:
Any particular reason why you removed the curly brackets here? They are especially useful in situations where they are string interpolated and you need to distinguish between the variable name and the rest of the string. E.g. `"$helloworld"` vs `"${hello}world"`. This just happen to work because there is a `-` character in `$prefix-kubeconfig`.

Reply:
Not sure why this got changed if I am being honest, I added them back.